resources/tentative/todomvc/src/components/todo-bottombar/todo-bottombar.component.js

```diff
@@ -0,0 +1,96 @@
+import template from "./todo-bottombar.template.js";
+
+class TodoBottombar extends HTMLElement {
+    static get observedAttributes() {
+        return ["total-items", "active-items"];
+    }
+
+    constructor() {
+        super();
+
+        // elements
+        this.element = undefined;
+        this.clearCompletedButton = undefined;
+        this.todoStatus = undefined;
+        this.filterLinks = undefined;
+        // state
+        this.connected = false;
```

Comment:
Why do we need this? We can just use [Node.prototype.isConnected](https://developer.mozilla.org/en-US/docs/Web/API/Node/isConnected)?